### DISCUSSION TITLE
Any suggestion to replace deprecated tinymce.create() api?

### DISCUSSION THREAD

**Discussion by Seungchul-de** [tinymce/tinymce#9505] [General]
Hi Expert of tinymce,

Recently, I am working on migrating from version 5 to 6
and found that tinymce.create() is deprecated. 

Currently, we are using this for customizing existing open-source image plugins, but after losing this API, I have no idea how to replace it.
Do you have any good suggestion to replace it simply? 
or Should I start from scratch from image plugin?

Thanks in advance

**Comment by TheSpyder** [comment]
`create` was part of an older class-based architecture. How were you using it?

**Reply by Seungchul-de** [reply]
current implementation is 
1. create a custom plug-in class with "create" api 
2. register event for beforeOpenWindow and AfterOpenWindow.
3. any plug-in is opened, this class check dialog identity and if dialog is image plug-in, then modify dialog to customDialog.
and also replace change and submit handler to customer.  

Do you have any suggestion?

**Reply by TheSpyder** [reply]
I don't think you need to use the `create` API for that, plugins are just regular objects now.
